apps/ssr-tests/package.json

```diff
@@ -15,9 +15,9 @@
   "devDependencies": {
     "@fluentui/react": "^8.104.6",
     "@microsoft/load-themed-styles": "^1.10.26",
-    "@types/mocha": "^7.0.2",
+    "@types/mocha": "7.0.2",
```

Comment:
Should these still be carets instead of exact matches?

Reply:
not at the moment, In upcoming weeks all devDeps will be migrated to exist only in root package.json https://github.com/microsoft/fluentui/issues/21395#issuecomment-1400137647